packages/core/src/libraries/hook/index.ts

```diff
@@ -54,6 +36,53 @@ export const createHookLibrary = (queries: Queries) => {
     hooks: { findAllHooks, findHookById },
   } = queries;
 
+  const sendWebhooks = async <T extends HookEventPayloadWithoutHookId>(hooks: Hook[], payload: T) =>
```

Comment:
should have some comments, especially it will insert logs while the name doesn't suggest (we didn't add comments, but better to do it now)